tests/wpunit/Tribe/Tickets/Ticket_ObjectTest.php

```diff
@@ -0,0 +1,595 @@
+	}
+
+	/**
+	 * @test
+	 * it should return the original stock/capacity
+	 *
+	 * @covers original_stock
+	 */
+	public function it_should_return_the_original_stock() {
+		$rsvp = $this->make_rsvp(
+			[
+				'meta_input' => [
+					'_capacity'   => 10,
+					'_stock'      => 5,
+					'total_sales' => 5,
+				],
+			]
+		);
+
+		$this->assertEquals( 10, $rsvp->original_stock() );
+
+		$ticket = $this->make_ticket(
+			1,
+			[
+				'meta_input' => [
+					'_capacity'   => 10,
+					'_stock'      => 5,
+					'total_sales' => 5,
+				],
+			]
+		);
+
+		$this->assertEquals( 10, $ticket->original_stock() );
+	}
+
+	/**
+	 * @test
+	 * it should return correct current inventory
+	 * Note: inventory is based on attendees! So we can't just manually set it (as stock)
+	 *
+	 * @covers inventory
+	 */
+	public function it_should_return_correct_current_inventory() {
+		$rsvp = $this->make_rsvp(
+			[
+				'meta_input' => [
+					'_capacity'   => 10,
+				],
+			]
+		);
+
+		$this->create_many_attendees_for_ticket( 5, $rsvp->ID, $rsvp->get_event_id() );
+
+		$this->assertEquals( 5, $rsvp->inventory(), 'Incorrect inventory reported for RSVP.' );
+
+		$ticket = $this->make_ticket(
+			1,
+			[
+				'meta_input' => [
+					'_capacity'   => 10,
+				],
+			]
+		);
+
+		$this->create_many_attendees_for_ticket( 5, $ticket->ID, $ticket->get_event_id() );
+
+		$this->assertEquals( 5, $ticket->inventory(), 'Incorrect inventory reported for Ticket.' );
+	}
+
+	/**
+	 * @test
+	 * it should return correct "own" capacity
+	 *
+	 * @covers capacity
+	 */
+	public function it_should_return_correct_own_capacity() {
+		$rsvp = $this->make_rsvp(
+			[
+				'meta_input' => [
+					'_capacity'   => 10,
+				],
+			]
+		);
+
+		$this->assertEquals( 10, $rsvp->capacity(), 'Incorrect capacity reported for new RSVP.' );
+
+		$this->create_many_attendees_for_ticket( 5, $rsvp->ID, $rsvp->get_event_id() );
+
+		$this->assertEquals( 10, $rsvp->capacity(), 'Incorrect capacity reported for RSVP with attendees.' );
+
+		$ticket = $this->make_ticket(
+			1,
+			[
+				'meta_input' => [
+					'_capacity'   => 10,
+				],
+			]
+		);
+
+		$this->assertEquals( 10, $ticket->capacity(), 'Incorrect capacity reported for new ticket.' );
+
+		$this->create_many_attendees_for_ticket( 5, $ticket->ID, $ticket->get_event_id() );
+
+		$this->assertEquals( 10, $ticket->capacity(), 'Incorrect capacity reported for ticket with attendees.' );
+	}
+
+	/**
+	 * @test
+	 * it should return correct "unlimited" capacity
+	 *
+	 * @covers capacity
+	 */
+	public function it_should_return_correct_unlimited_capacity() {
+		$rsvp = $this->make_rsvp(
+			[
+				'meta_input' => [
+					'_capacity'   => -1,
+				],
+			]
+		);
+
+		$this->assertEquals( -1, $rsvp->capacity(), 'Incorrect capacity reported for new RSVP.' );
+
+		$this->create_many_attendees_for_ticket( 5, $rsvp->ID, $rsvp->get_event_id() );
+
+		$this->assertEquals( -1, $rsvp->capacity(), 'Incorrect capacity reported for RSVP with attendees.' );
```

Comment:
Maybe `Incorrect capacity reported for *unlimited* RSVP with attendees.`? (to be a bit clearer)
Same for ticket below.